### PR TITLE
Fixed 'Bug 57352 - Parentheses closing is broken when you do an if

### PR DESCRIPTION
**Pull request by mkrueger**
statement.'

I joined the two auto insert bracket settings and removed the add only
opening bracket option for completion.

### CODE REVIEW

File: main/tests/UnitTests/MonoDevelop.CSharpBinding/NamedArgumentCompletionTests.cs [hunk 1]
```diff
@@ -218,11 +218,11 @@ public void Replace (int offset, int count, string text)
 			return widget.CompletedWord;
 		}
 
-		
+		// 0172 3760533
```
Comment (KirillOsenkov) [inline main/tests/UnitTests/MonoDevelop.CSharpBinding/NamedArgumentCompletionTests.cs]: What is this? :)

File: main/src/addins/MonoDevelop.SourceEditor2/MonoDevelop.SourceEditor.OptionPanels/CompletionOptionsPanel.cs [hunk 2]
```diff
@@ -65,9 +65,9 @@ Control IOptionsPanel.CreatePanelWidget ()
 			includeCodeSnippetsCheckbutton.Active = IdeApp.Preferences.IncludeCodeSnippetsInCompletionList;
 			automaticCompletionModeCheckbutton.Active = !IdeApp.Preferences.ForceSuggestionMode;
 
-			insertParenthesesCheckbutton.Active = IdeApp.Preferences.AddParenthesesAfterCompletion;
-			openingRadiobutton.Active = IdeApp.Preferences.AddOpeningOnly;
-			bothRadiobutton.Active = !IdeApp.Preferences.AddOpeningOnly;
+			insertParenthesesCheckbutton.Visible = false;
```
Comment (DavidKarlas) [inline main/src/addins/MonoDevelop.SourceEditor2/MonoDevelop.SourceEditor.OptionPanels/CompletionOptionsPanel.cs]: Can we delete this controls and not just hide them?

**Comment by slluis** [comment]
Please create a build lane for this PR to verify nothing breaks before merging.

File: main/src/addins/MonoDevelop.SourceEditor2/MonoDevelop.SourceEditor.OptionPanels/CompletionOptionsPanel.cs (hunk 2 above)
Comment (mkrueger) [inline main/src/addins/MonoDevelop.SourceEditor2/MonoDevelop.SourceEditor.OptionPanels/CompletionOptionsPanel.cs]: Stetic was broken for me - didn't load the file cleanly. But anyways it's not visible, and if I need to add new check buttons I can just reuse these.

Comment (Therzok) [inline main/src/addins/MonoDevelop.SourceEditor2/MonoDevelop.SourceEditor.OptionPanels/CompletionOptionsPanel.cs]: > Stetic was broken for me

Fixing!

**Comment by mkrueger** [comment]
https://code.internalx.com/Wrench/BuildDetails?buildId=a0a1f39fe97982411fe21ddf3f7998cf

**Comment by slluis** [comment]
Can you please rebase to latest master so that we can get a green build?

**Comment by slluis** [comment]
Ping.

**Comment by mkrueger** [comment]
Any news on that one ?

**Comment by DavidKarlas** [comment]
I approved the change, afaiu since you rebased and it built successfully and there are only few tests failing which are unrelated, so imho, its good to merge

**Pull request merged**